imagej/imagej-ops2-tutorial/docs/MigratingFromImageJOps.md

```diff
@@ -0,0 +1,198 @@
+# Migrating from ImageJ Ops
+
+The [ImageJ Ops](https://imagej.net/libs/imagej-ops/index) framework is the predecessor of the SciJava Ops framework, and many steps were taken to make expressing Ops much easier. This document shows ImageJ Ops developers how to convert their Ops to SciJava Ops - we'll convert [`DefaultGaussRAI`](https://github.com/imagej/imagej-ops/blob/master/src/main/java/net/imagej/ops/filter/gauss/DefaultGaussRAI.java) from the ImageJ Ops repository as an example.
+
+## 1. Upgrading pom-scijava
+
+Before doing anything else, you should ensure that your version of pom-scijava is new enough to make use of SciJava 3 goodness.
+
+TODO: Replace with the pom-scijava version needed to grab this annotation processor.
+
+Until the SciJava 3 annotation processor is added to pom-scijava, developers must add the following block of code to the `build` section of your POM:
+
+```xml
+<build>
+    <plugins>
+        <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+                <annotationProcessorPaths>
+                    <path>
+                        <groupId>org.scijava</groupId>
+                        <artifactId>scijava-javadoc-parser</artifactId>
+                        <version>${project.version}</version>
+                    </path>
+                </annotationProcessorPaths>
+                <fork>true</fork>
+                <showWarnings>true</showWarnings>
+                <compilerArgs>
+                    <arg>-Ajavadoc.packages="${therapi.packages}"</arg>
+                </compilerArgs>
+            </configuration>
+        </plugin>
+    </plugins>
+</build>
+```
```

Comment:
Does this all need to change after https://github.com/scijava/incubator/pull/79?

Reply:
Yeah, it will have to change, in one of two ways:
1. We move this block upstream, like we did for [therapi](https://github.com/scijava/pom-scijava-base/pull/31) - in this case, we'll probably need to add some opt-in flag to POMs.
2. We don't move this block upstream, and we simply change the annotation processor to use the new `scijava-javadoc-parser` module, like we do in that PR.

Either way, we should either (a) not merge this until #79 merges, or (b) file an issue to change it later.